src/harness/sas_testcase.py

```diff
@@ -165,4 +165,33 @@ def assertChannelsContainFrequencyRange(self, channels, frequency_range):
             
     channels.sort(key=lambda ch: (ch['frequencyRange']['highFrequency']), reverse = True)
     self.assertEqual(channels[0]['frequencyRange']['highFrequency'],\
-             frequency_range['highFrequency'])
\ No newline at end of file
+             frequency_range['highFrequency'])
+   
+  def TriggerFullActivityDumpAndWaitUntilComplete(self, server_cert, server_key):
+    """ this function trigger SAS UUT to generate dump files \
+    and wait for the generation of them with a timeout of 7200 seconds
+    
+    Args: the certificate and the key of the server
+    
+    return : dump message"""
+    self._sas_admin.TriggerFullActivityDump()
+    request_time = datetime.utcnow()
```

Comment:
Please put this before the FAD trigger.

Reply:
yes sure, I don't why I put it here I updated the code.